FindBaselibs.cmake

```diff
@@ -1,10 +1,61 @@
-set (BASEDIR "" CACHE PATH "Path to installed baselibs _including_ OS subdirectory (Linux or Darwin).")
-set (Baselibs_FOUND FALSE)
+set (BASEDIR "" CACHE PATH "Path to installed baselibs")
+set (Baselibs_FOUND FALSE CACHE BOOL "Baselibs Found")
```

Comment:
Technically this could be done with `option` but  I think it is better this way, as it strongly implies not to use the command line to override.